### PR TITLE
Make the description more clear

### PR DESCRIPTION
**Pull request by xauthulei**
It is a bit confuse to introduce the tool `kind`.  Thanks

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]

File: hack/README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ See [DEVELOPMENT.md](https://github.com/tektoncd/plumbing/blob/main/DEVELOPMENT.
 
 ### tekton_in_kind.sh
 
-This script uses the [`kind` tool](https://kind.sigs.k8s.io/) to create a local K8s cluster in Docker and then deploys [Tekton Pipeline](https://github.com/tektoncd/pipeline), [Tekton Triggers](https://github.com/tektoncd/triggers) and [Tekton Dashboard](https://github.com/tektoncd/dashboard) components, into it.
+This script uses the [`kind`](https://kind.sigs.k8s.io/) to create a local K8s cluster in Docker and then deploys [Tekton Pipeline](https://github.com/tektoncd/pipeline), [Tekton Triggers](https://github.com/tektoncd/triggers) and [Tekton Dashboard](https://github.com/tektoncd/dashboard) components, into it.
```
Comment (dibyom) [inline hack/README.md:20]:
```suggestion
This script uses [`kind`](https://kind.sigs.k8s.io/) to create a local K8s cluster in Docker and then deploys [Tekton Pipeline](https://github.com/tektoncd/pipeline), [Tekton Triggers](https://github.com/tektoncd/triggers) and [Tekton Dashboard](https://github.com/tektoncd/dashboard) components, into it.
```

Super minor grammatical nit

**Comment by xauthulei** [comment]
@dibyom Thanks for your review comments.

**Review by savitaashture — APPROVED** [review]
/lgtm

**Pull request merged**